homeassistant/components/switch/sony_projector.py

```diff
@@ -0,0 +1,102 @@
+"""
+Use SDCP network control to power on/off and check state of Sony projector.
+
+For more details about this component, please refer to the documentation
+at https://home-assistant.io/components/switch.sony_projector/
+
+Example configuration:
+
+switch:
+  - platform: sony_projector
+    host: "192.168.1.43"
```

Comment:
Example configuration should be moved to the documentation.

Reply:
Removed. I'd read in the style guide that basic config should in the comment header, which is why it was there. PR for doc updates to follow.